### PR TITLE
New filepath for rapid files

### PR DESCRIPTION
**Pull request by RosalynLP**
Updating filepath and filename for the new rapid files.

### CODE REVIEW

File: Dashboard Data Transfer/Transfer scripts/transfer_admissions.R [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
 
 read_rds_with_options <- create_loader_with_options(readRDS)
-i_chiadm <- read_rds_with_options(glue("{adm_path}/Pubs {format(report_date-2, format='%d%m')}/CHI_Admissions_Positives_{format(report_date-2, format='%Y-%d-%m')}.rds"))
+i_chiadm <- read_rds_with_options(glue("{adm_path}/CHI_Admissions_Positives_provisional.rds"))
 
 o_adm_simd <- read.csv(glue("{output_folder}/Admissions_SIMD.csv"), header = TRUE, stringsAsFactors = FALSE, check.names=FALSE)
 
 # Filter CHI and 12 files down to last Tuesday
-i_chiadm %<>%filter(admission_date <= (report_date - 8))
+i_chiadm %<>% filter(admission_date <= (report_date - 8))
```
Comment (rubenv95) [inline Dashboard Data Transfer/Transfer scripts/transfer_admissions.R:16]: Will need to change this to be the most recent Sunday

**Comment by rubenv95** [comment]
I'm going to speak to Lynsey about finalise some text that can be added to the dashboard explaining the move to weekly.

File: Dashboard Data Transfer/Transfer scripts/transfer_admissions.R (hunk 1 above)
Comment (RosalynLP) [reply Dashboard Data Transfer/Transfer scripts/transfer_admissions.R:16]: so report_date -3?

Comment (RosalynLP) [inline Dashboard Data Transfer/Transfer scripts/transfer_admissions.R:16]:
```suggestion
i_chiadm %<>% filter(admission_date <= (report_date - 3))
```

Comment (RosalynLP) [reply Dashboard Data Transfer/Transfer scripts/transfer_admissions.R:16]: @rubenv95 was this what you mean?

Comment (rubenv95) [reply Dashboard Data Transfer/Transfer scripts/transfer_admissions.R:16]: Yes thanks, can you also please add a "p" for provisional to the latest week of data for hospital admissions data tables?

**Review by rubenv95 — APPROVED** [review]
Your changes look good. 

I've added to the notes tab, changed the admission date week ending to Sun, and remove the prop/cases stuff.

Everything except for "Distribution of COVID-19 cases by age group" table seems to be working still. We can create that output with lab data on Monday and update the data transfer script accordingly. 
Once you're happy with my changes you can merge!

**Pull request merged**